Libraries/Text/TextInput/RCTBaseTextInputViewManager.m

```diff
@@ -74,6 +74,8 @@ @implementation RCTBaseTextInputViewManager {
 RCT_EXPORT_VIEW_PROPERTY(clearTextOnSubmit, BOOL);
 RCT_EXPORT_VIEW_PROPERTY(onSubmitEditing, RCTBubblingEventBlock);
 RCT_EXPORT_VIEW_PROPERTY(submitKeyEvents, NSArray<NSDictionary *>);
+
+RCT_REMAP_OSX_VIEW_PROPERTY(cursorColor, backedTextInputView.cursorColor, UIColor)
```

Comment:
Nit: Now that it's in a macOS ifdef block, we can just use `RCT_EXPORT_VIEW_PROPERTY`.

Reply:
Do you mean `RCT_REMAP_VIEW_PROPERTY`?

Reply:
... right, good catch!